_layouts/blog.html

```diff
@@ -37,16 +37,18 @@ <h1><a href="/blog/">Trino blog</a></h1>
   </div>
 
   <!-- Comment to remove TOC from a page -->
-  {% if content contains '<!-- No TOC -->' %}
+  {% if content contains '<!-- No TOC -->' or page.show_pagenav==false %}
```

Comment:
You can probably do a little refactor here and remove the `<!-- No TOC -->` and add `page.show_pagenav==false` where it exists to clean things up.

I also think it'd be better to assume we _don't_ want a TOC, only display it when the property is set to true, and then manually add it to blogs and pages where we think it would be useful. But that can be a later bit of cleanup.

Reply:
This should be done in a follow up PR .. the default can be declared in the global config file.